### ISSUE TITLE
Next update?

### ISSUE THREAD

**Issue by EfronC** [Floens/Clover#794]
This is not an Issue itself, and i didn't want to put it here, but since i didn't receive an answer from the contact mail, and 2 weeks have passed since, i'll put it anyway: Is there an estimate for when next release will be launched? I saw the there has been a lot of commits with important updates and fixes, but the last release goes back to March 2019(3.0.2), more than one year ago, and nor the inner updater, nor F-Droid detect new updates.

I know about Clover Dev, which updates with every commit, but for some reason, the version of it i've on my phone stopped updating, every time i click "Search Updates", tells me there are, start downloading, fails, and doesn't do anything, and since i don't really want to delete the app every time it fails, mostly because of all the configs(Filters, Sites) and Markers from old threads that i open on archive sites(Maybe a good feature would be a way to save all those configs and markers on a file so you can import them when reinstalling?).

**Comment by TacoTheDank** [comment]
@EfronC You could manually download the latest Dev build from https://chandevel.github.io/Clover/#dev to get around the failing downloads.

**Comment by EfronC** [comment]
@TacoTheDank Yeah, but that would require to reinstall the app, and as i mentioned, i've a lot of things saved on the app that can't be backed up(There's not feature for that), that's why i haven't taken that option as an answer, otherwise i would be using Clover Dev.

**Comment by TacoTheDank** [comment]
@EfronC Figured that would be the response. I don't think there's anything you can really do, unfortunately, but I'd recommend [Kuroba](https://github.com/Adamantcheese/Kuroba) as a current and improved replacement for Clover.

One of Kuroba's many features is a backup feature.

**Comment by EfronC** [comment]
@TacoTheDank Have been using Kuroba for a few days to try, and first impression is that it has MANY QoL features that Clover desperately needs, but i also noted how the app crashed very oftenly, and has very important bugs/errors, like putting messages in transparent background with black font, causing the text being impossible to read. From this i can deduce that the app is still incredible unstable, so, while Kuroba might be a good alternative, probably still needs some baking time. Thanks for the recommendation by the way :D

**Comment by TacoTheDank** [comment]
@EfronC You're welcome. While the app has a few issues that need ironing out, the dev is very responsive to issues and fixes bugs very often. You can always create issues for these in the Kuroba repository. :P

(Also, that's kinda strange. My Kuroba rarely crashes, and barely has any bugs or errors. Maybe it's just because I'm on Android 10 lol.)

One other thing: a big contributor to the Kuroba repository is also working on their own fork/redesign of Kuroba [here](https://github.com/K1rakishou/Kuroba-Experimental). I recommend also trying that out if you want.

**Comment by i-cant-git** [comment]
Transparent background issue was fixed in [#1018](https://github.com/Adamantcheese/Kuroba/issues/1018), I asked the dev to make a new release but he hasn't responded. Also I agree that it's very stable, never crashed once. (I use Android 7)
